eslint-config-opent2t/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "eslint-config-opent2t",
```

Comment:
We've been *starting* our package names with "opent2t-"
So maybe name this "opent2t-config-eslint" ?